src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -52,7 +52,7 @@ namespace System.Management.Automation
     ///     - There is no ITypeDescriptorContext.
```

Comment:
@iSazonov, your last commit had 60 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

TypeTable duplicate types in reused runspace InitialSessionState TypeTable.No duplicate types test.Error occurred in Context block
```powershell
Multiple ambiguous overloads found for "Add" and the argument count: "1".
At /Users/runner/runners/2.169.1/work/1/s/test/powershell/engine/Api/InitialSessionState.Tests.ps1:72 char:13
```
TypeTable duplicate types in reused runspace InitialSessionState TypeTable.Cannot use shared TypeTable in ISS test.Error occurred in Context block
```powershell
Multiple ambiguous overloads found for "Add" and the argument count: "1".
At /Users/runner/runners/2.169.1/work/1/s/test/powershell/engine/Api/InitialSessionState.Tests.ps1:105 char:13
```
Task-based PowerShell async APIs.PowerShell::InvokeAsync - Single script tests.can invoke a single script asynchronously, but only in a new runspace
```powershell
Cannot convert argument "tasks", with value: "System.Object[]", for "WaitAll" to type "System.Threading.Tasks.Task": "Cannot convert the "System.Object[]" value of type "System.Object[]" to type "System.Threading.Tasks.Task"."
at <ScriptBlock>, /Users/runner/runners/2.169.1/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1: line 28
28:                     [System.Threading.Tasks.Task]::WaitAll(@($r))

```
Task-based PowerShell async APIs.PowerShell::InvokeAsync - Single script tests.handles terminating errors properly when invoked asynchronously in a new runspace
```powershell
Expected an exception, with FullyQualifiedErrorId 'AggregateException' to be thrown, but the FullyQualifiedErrorId was 'MethodArgumentConversionInvalidCastArgument'. from /Users/runner/runners/2.169.1/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1:70 char:21
    +                     [System.Threading.Tasks.Task]::WaitAll(@($r))
    +                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.169.1/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1: line 74
74:                 { $sb.Invoke() } | Should -Throw -ErrorId 'AggregateException'
```
Task-based PowerShell async APIs.PowerShell::InvokeAsync - Single script tests.cannot invoke a single script asynchronously in a runspace that is busy
```powershell
Expected an exception, with FullyQualifiedErrorId 'AggregateException' to be thrown, but the FullyQualifiedErrorId was 'MethodArgumentConversionInvalidCastArgument'. from /Users/runner/runners/2.169.1/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1:28 char:21
    +                     [System.Threading.Tasks.Task]::WaitAll(@($r))
    +                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.169.1/work/1/s/test/powershell/engine/Api/TaskBasedAsyncPowerShellAPI.Tests.ps1: line 116
116:                     $err = { InvokeAsyncHelper -PowerShell $ps2 -Wait } | Should -Throw -ErrorId 'AggregateException' -PassThru
```